packages/shared/src/domain/pagination.ts

```diff
@@ -34,7 +34,7 @@ const queryMetaSchema = createQueryMetaSchema();
 export type HttpMeta = z.infer<typeof queryMetaSchema>;
 
 export const paginationMetaSchema = z.object({
-  itemsInTotal: z.number(),
+  itemsInTotal: z.number().optional(),
```

Comment:
❗  Why did we want to change this? This has big ramifications on typesafety of our api

Reply:
@lucasdellabella It was possible to receive a page where it's not being set, which was causing the schema validation for roster upload to fail - maybe needs a fix on the API side.